test/EVMHost.cpp

```diff
@@ -1117,6 +1119,12 @@ evmc::Result EVMHost::precompileALTBN128PairingProduct(evmc_message const& _mess
 	return precompileGeneric(_message, inputOutput);
 }
 
+evmc::Result EVMHost::precompileBlake2f(evmc_message const&) noexcept
+{
+	// TODO implement
```

Comment:
Is it not implemented here because there are no tests that use `blake2` so there's nothing to do? If so, we should probably add some. How about one of the test vectors from the EIP?

Reply:
Blake2 is not used, it would be a waste of time implementing it here imho.

The "code" is the same as for `EVMHost::precompileModExp`.

Reply:
ok, I guess it's fine then.

Reply:
We can add a test for both blake2 + modexp, but it may take a bit for me to do it, i.e. if you could add 1-1 simple test case for it that would be appreciated.

It has to be in inline assembly though because they are not exposed in the language.